### PR TITLE
py3dns : initial commit

### PR DESCRIPTION
**Pull request by D4imyo**
a tool dependency

### CODE REVIEW

File: packages/py3dns/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-dns
```
Comment (noraj) [inline packages/py3dns/PKGBUILD:4]:
```diff
-pkgname=python-dns
+pkgname=python-py3dns
```

File: packages/py3dns/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+_pkgname=py3dns
+pkgver=3.2.1
+pkgrel=1
+pkgdesc='A module for looking up DNS entries in Python applications.'
+arch=('any')
+url='https://pypi.org/project/py3dns/'
+license=('Python')
```
Comment (noraj) [inline packages/py3dns/PKGBUILD:11]: Python License (CNRI Python License) is not in `/usr/share/licenses/common/`

please follow https://wiki.archlinux.org/title/PKGBUILD#license

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/py3dns/PKGBUILD (hunk 1 above)
Comment (D4imyo) [reply packages/py3dns/PKGBUILD:4]: dose this applies on any python library with name like "pyfoo" ?

File: packages/py3dns/PKGBUILD (hunk 2 above)
Comment (D4imyo) [reply packages/py3dns/PKGBUILD:11]: thanks, totally forgot it :))

File: packages/py3dns/PKGBUILD (hunk 1 above)
Comment (noraj) [reply packages/py3dns/PKGBUILD]: Yes.
You have example in AL, if a Pypi package is named `python-multipart` then the packake name should be `python-python-multipart`.
https://archlinux.org/packages/community/any/python-python-multipart/
https://archlinux.org/packages/community/x86_64/python-python-pkcs11/

File: packages/py3dns/PKGBUILD [hunk 3]
```diff
@@ -24,5 +24,7 @@ package() {
   cd "$_pkgname-$pkgver"
 
   python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+  
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$_pkgname-$pkgver/LICENSE"
```
Comment (noraj) [inline packages/py3dns/PKGBUILD:28]: `install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"` do not include the version

**Review by noraj — CHANGES_REQUESTED** [review]

**Comment by noptrix** [comment]
@D4imyo thanks!

**Comment by noptrix** [comment]
@noraj thank you!

**Pull request merged**